### ISSUE TITLE
Main feed shouldn't be a direct child to root feed

### ISSUE THREAD

**Issue by Powersource** [ssbc/ssb-meta-feeds-spec#35]
Currently a few of the specs make it sound like that

**Comment by arj03** [comment]
Yes, I'm trying to move some of this over here, and only have this spec about metafeeds:

https://github.com/ssbc/ssb-meta-feeds-migration
